blows up when running inspect-buildpack without arguments

fyi, I was just playing around with this, and noticed that the command blows up when running inspect-buildpack without arguments. It should print the help text.

```
./pack inspect-buildpack
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/buildpacks/pack/internal/commands.InspectBuildpack.func1(0xc000336000, 0x1f53d70, 0x0, 0x0, 0x0, 0x11de7e0)
	github.com/buildpacks/pack/internal/commands/inspect_buildpack.go:69 +0x421
github.com/buildpacks/pack/internal/commands.logError.func1(0xc000336000, 0x1f53d70, 0x0, 0x0, 0x0, 0x0)
	github.com/buildpacks/pack/internal/commands/commands.go:53 +0x6c
github.com/spf13/cobra.(*Command).execute(0xc000336000, 0x1f53d70, 0x0, 0x0, 0xc000336000, 0x1f53d70)
	github.com/spf13/cobra@v0.0.6/command.go:840 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0xc0000cd340, 0xc00033c140, 0xc0000cd340, 0x0)
	github.com/spf13/cobra@v0.0.6/command.go:945 +0x317
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v0.0.6/command.go:885
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	github.com/spf13/cobra@v0.0.6/command.go:878
main.main()
	github.com/buildpacks/pack/cmd/pack/main.go:26 +0xeb
```

The command is great though, and will be super helpful to users! I really like the addition of the depth flag

_Originally posted by @kvedurmu in https://github.com/buildpacks/pack/issues/642#issuecomment-696994250_

--- a/internal/commands/inspect_buildpack.go
+++ b/internal/commands/inspect_buildpack.go
@@ -64,7 +64,7 @@ func InspectBuildpack(logger logging.Logger, cfg *config.Config, client PackClie
 	cmd := &cobra.Command{
 		Use:   "inspect-buildpack <image-name>",
 		Short: "Show information about a buildpack",
-		Args:  cobra.MaximumNArgs(4),
+		Args:  cobra.RangeArgs(1, 4),
 		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
 			buildpackName := args[0]
 			registry := flags.Registry

--- a/internal/commands/inspect_buildpack_test.go
+++ b/internal/commands/inspect_buildpack_test.go
@@ -365,7 +365,7 @@ func testInspectBuildpackCommand(t *testing.T, when spec.G, it spec.S) {
 					}).Return(simpleInfo, nil)
 				})
 
-				it.Focus("succeeds", func() {
+				it("succeeds", func() {
 					command.SetArgs([]string{"test/buildpack"})
 					assert.Nil(command.Execute())
 
@@ -636,15 +636,15 @@ func testInspectBuildpackCommand(t *testing.T, when spec.G, it spec.S) {
 				err := command.Execute()
 
 				assert.Error(err)
-				assert.Contains(err.Error(), "error inspecting remote archive")
+				assert.Contains(err.Error(), "error writing buildpack output: \"no buildpacks found\"")
 			})
 		})
 
 		when("unable to inspect buildpack on registry", func() {
 			it.Before(func() {
 				mockClient.EXPECT().InspectBuildpack(pack.InspectBuildpackOptions{
-					BuildpackName: "image-failure-case/buildpack",
-					Daemon:        false,
+					BuildpackName: "urn:cnb:registry:registry-failure/buildpack",
+					Daemon:        true,
 					Registry:      "some-registry",
 				}).Return(&pack.BuildpackInfo{}, errors.New("error inspecting registry image"))
 			})

